develop/TOOLS/NuGet/Consuming_NuGet.md

```diff
@@ -4,44 +4,41 @@ uid: Consuming_NuGet
 
 > [!IMPORTANT]
-> DIS currently only processes the *lib* folder of NuGet packages. Other folders such as *ref* or *runtimes* are currently not supported.
+>
+> - DIS currently only processes the *lib* folder of NuGet packages. Other folders such as *ref* or *runtimes* are currently not supported.
+> - In a connector or Automation script solution, you cannot use NuGet packages that generate code (for example the [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools) NuGet package). This is because when DataMiner compiles a connector or Automation script, it only considers the C# code that is included in the XML file of the connector or Automation script. Therefore, if you want to make use of this NuGet package, you need to include the generated code in the connector or Automation script solution.
+> - When consuming different versions of the same NuGet package, make sure you are aware of the potential pitfalls when doing so as explained in <xref:Run_Time_Assembly_Binding>.
 
 Within Skyline, we have an internal NuGet store (<https://devcore3/nuget>) where Skyline employees can produce and consume private libraries. You will need to [add this store in Visual Studio](https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio#package-sources) to use it.
 
-## Using the Class Library as a NuGet
-
-The Class Library (Common, Protocol and Automation code) is also available and ready to be used as a NuGet in the internal store.
-
-It is possible to use this instead of DIS Code Generation. This can help solve some of the common issues that occur with DIS Code Generation, such as OS maximum path length problems. It will also allow you to more easily make your own libraries on top of the class library.
-
-> [!NOTE]
-> There is a breaking change for these versions of the Class Library. InterApp Calls are no longer able to use reflection. This makes their use much more stable, but it does mean that you always need to provide a list of known types for every call or serialize/deserialize operation. Upgrading to these versions will likely cause compilation issues that need to be fixed. However, you can now put all your Message classes into a custom solution and re-use that as a NuGet of your own creation.
-
 ## Licensing
 
-NuGet Libraries from nuget.org are considered third-party libraries, so make sure licensing is OK.
+NuGet Libraries from [nuget.org](https://nuget.org) are considered third-party libraries, so make sure licensing is OK.
 
-Consult the Stop/Go/Caution list on the internal Skyline wiki. If your license does not appear on the list, or if the use you intend (i.e. internal vs. distribution) is not listed for that license, the use will be considered a "Stop".
+Consult the Stop/Go/Caution list on the internal Skyline Wiki. If your license does not appear on the list, or if the use you intend (i.e. internal vs. distribution) is not listed for that license, the use will be considered a "Stop".
```

Comment:
Wiki doesn't exist anymore. Refer to Sharepoint (https://skylinebe.sharepoint.com/sites/LegalPortal/SitePages/Stop-Go-Caution-List.aspx). Probably only accessible internally though.

Reply:
updated